### ISSUE TITLE
Apply design to Edit Username functionality

### ISSUE THREAD

**Issue by churik** [status-im/status-react#5935]
### Description

*Type*: Bug

*Summary*: make Edit Username functionality in Profile according to design in https://github.com/status-im/status-react/issues/5456 (zpl.io/brE1NdX).
Created in continuation of https://github.com/status-im/status-react/pull/5896

### Summary: 

- [ ] Put focus to the editable area when the user is in editable mode

When you open Profile -> Edit you should manually set focus to username field. Now it is looking a bit confusing, especially when area is not distinguished from background.
I'm suggesting to set focus automatically to username area when you click on "Edit" (like it is made for mobile)
Video: http://take.ms/8A5ok

*State:* 
blocked by RN issue: https://github.com/status-im/react-native-desktop/issues/338

- [ ]  Input area is not aligned according to design.
![px](https://user-images.githubusercontent.com/4557972/45752722-1e1c9d80-bc1f-11e8-8c8a-59a098b950e1.png)

- [ ] Font doesn't become italic

- [ ] Username is cut in editable mode when it contains emoji
![emoji](https://user-images.githubusercontent.com/4557972/45753475-b287ff80-bc21-11e8-9947-a675b569306a.png)

### Additional Information
* Status version: 21/09/2018
* Operating System: Mac OsX

<blockquote><img src="https://api.monosnap.com/rpc/file/download?id=ij05ZjVLY8wiAa8r3r7s9WvUaecXFr&type=preview" width="48" align="right"><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/ij05ZjVLY8wiAa8r3r7s9WvUaecXFr">File "screencast 2018-09-19 14-53-21.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by churik** [comment]
@EugeOrtiz review is appreciated, probably I missed smth.

**Comment by churik** [comment]
Closed as duplicate of https://github.com/status-im/status-react/issues/5933
